runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -240,4 +240,18 @@ export class RepeatingFieldPageController extends PageController {
     const partial = this.getPartialState(state) ?? [];
     return partial.length;
   }
+
+  covertMultiInputStringAnswers(answers) {
+    for (let i = 0; i < answers.length; i++) {
+      if (typeof answers[i] === "string") {
+        const values = answers[i].split(":");
+        let multiInput = {
+          "type-of-revenue-cost": values[0].trim(),
```

Comment:
Is there any way not to hardcode this field name? Will this make it only work in this one instance?

Reply:
Have discussed offline with @adamdavies1 we agree this is sub-optimal but necessary for now (and we already have this hardcoded in other places.) Suggested to make a backlog ticket to improve these instances.